templates/modules/b2b-account/quotes/quotes.hyper.live

```diff
@@ -28,18 +28,49 @@
     <h3 class="mz-l-stack-sectiontitle">{{ labels.quotes }}</h3>
     <div class="mz-b2b-quotes">
         <div class="mz-quotes-grid-container">
-            <div>
+            <div class="filter-container">
                 {% if user.isSalesRep %}
-                <div class="mz-b2b-space" ></div> 
-                {% endif %} 
-                <input type="text" name="searchName" id="searchName" data-mz-action="applyfilter" placeholder="Quote Name">
-                <Span class="mz-b2b-quotes-compare"> > </Span>
-                <input type="date" data-mz-is-datepicker id="expirationdate" data-mz-action="applyDatefilter" data-format="yy/mm/dd" placeholder="Search Exp Date">
+                <div>
+                    <select class="account-select" data-mz-action="selectionChanged" id="accountNameDropdown" data-mz-value="">
+                        <option selected="selected" value="">
+                            {{labels.accountSearchLabel}}
+                        </option>
+                        {% for acc in model.b2bAccounts %}
+                            {% if acc.data.id %}
+                            <option value="{{acc.data.id}}">
+                                {{acc.data.companyOrOrganization}}
+                            </option>
+                            {% endif %}
+                        {% endfor %}
+                    </select>
+                </div>
+                {% endif %}
+                <div>
+                    <input type="text" name="searchQuoteNumber" id="searchQuoteNumber" class="quotesTextSearch" data-mz-action="inputChanged" placeholder="Quote Number">
+                </div>
+                <div>
+                    <input type="text" name="searchName" id="searchName" class="quotesTextSearch" data-mz-action="inputChanged" placeholder="Quote Name">
+                </div>
+                <div class="exp-date-filter-container">
+                    <Span class="mz-b2b-quotes-compare"> > </Span>
+                    <input type="date" data-mz-is-datepicker id="expirationdate" class="expirationdate-filter" data-mz-action="dateChanged" data-format="yy/mm/dd" placeholder="Search Expiration Date">
+                </div>
+                <div class="fill-remaining-space"></div>
+                <div>
+                    <select class="status-select" data-mz-action="selectionChanged" id="statusDropdown" data-mz-value="">
+                    <option selected="selected" value="">{{ labels.filterStatus }}</option>           
+                    <option value="Pending">Pending </option>
```

Comment:
Can we ready dropdown options from json files??

Reply:
Done